couchdb/tests/client.py

```diff
@@ -145,9 +145,8 @@ def test_user_management(self):
         try:
             server.add_user('foo', 'secret', roles=['hero'])
             status, token = server.login_user('foo', 'secret')
-            self.assertEqual(status, 200)
-            self.assertTrue(server.verify_user(token))
-            self.assertTrue(server.logout_user(token))
+            server.verify_user(token)
```

Comment:
Why did you remove these? They seemed useful to me. The called methods can still return `False` on failure, right?